src/UNIV2LPOracle.sol

```diff
@@ -89,6 +98,7 @@ contract UNIV2LPOracleFactory {
         address tok1 = UniswapV2PairLike(_src).token1();
         orcl = address(new UNIV2LPOracle(_src, _wat, _orb0, _orb1));
         UNIV2LPOracle(orcl).rely(msg.sender);
```

Comment:
I'd suggest adding an owner param for relying the oracle. So the pause proxy can be set directly.
Like we did in the new fabs.

Reply:
If I understand correctly, you propose storing a variable called `owner` in the factory so that the Pause Proxy can be relied instead of `msg.sender`.

I just tried your suggestion, but denying the sender breaks many tests. Indeed, the testing contract needs to be able to call things like `daiEthLPOracle.kiss(address(this));`. Since `kiss` is authed and the testing contract isn't relied, it will fail.

Of course, it is still possible to `hevm.store` the authorization of the testing contract, but if we do that in `setUp`, then the test that checks the authorizations will fail.

Reply:
Storing the pause proxy is an option, the other is passing it via param in the `build` function (actually what I prefer).
The tests can be solved passing `address(this)` as the owner or storing it if we go with the other option.

Reply:
I like Gonzalo's suggestion--allow passing an address that will be `relied` instead of `msg.sender`.

Reply:
OK, I see. I will implement this ASAP.

Reply:
done.